conversion.py

```diff
@@ -1,5 +1,2 @@
 def celsius_to_kelvin(celsius):
-    return celsius + 273.15
-
-def celsius_to_fahrenheit(celsius):
-    return celsius * 9/5 + 32
\ No newline at end of file
+    return celsius + 273.15
```

Comment:
Not that great - do this instead
def celsius_to_fahrenheit(celsius):
    return celsius * 9/5 + 30